tests/test_basic.py

```diff
@@ -7,6 +7,7 @@
 import numpy as np
 import unittest
 import pickle
+import uuid
```

Comment:
uuid is standard library - https://docs.python.org/2/library/uuid.html (probably need to move the import up a few lines for style though)